pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -574,10 +580,7 @@ private static void setupCatchAllErrorHandling() {
         //JSON for Not Found (code 404) handling
         notFound((request, response) -> {
             LOG.info("[404] Current status: {}", response.status());
-            ApiError apiError = new ApiError(ErrorCodes.NOT_FOUND, "This page was not found.");
-            response.type(ContentType.APPLICATION_JSON.getMimeType());
-            SimpleJsonTransformer jsonTransformer = new SimpleJsonTransformer();
-            return jsonTransformer.render(apiError);
+            return ResponseUtil.renderPageNotFound(response);
```

Comment:
Refactored this out to ensure that our new whitelist filtering looks exactly likely our default  "page not found" functionality.